apps/examples/lcd_test/example_lcd.c

```diff
@@ -363,6 +425,25 @@ static void test_fps(void)
 	}
 }
 
+bool is_valid_power(char *power)
+{
+	int power_val_size = strlen(power);
+
+	if (power_val_size < 1 || power_val_size > 3) {		/* Length of Power val should be 1, 2, or 3 */
+		return false;
+	}
+	for (int i = 0; i < power_val_size; i++) {
+		if (!isdigit(power[i])) {	/* If not a digit */
+			return false;
+		}
+	}
```

Comment:
Is this necessary? If we call atoi directly, what happen? What can we prevent any wrong operation?

Reply:
if we enter input not integer char, the atoi will return 0. it will make lcd off.
for avoid it, this checking digital looks good.